foxpuppet/tests/test_browser_model.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from foxpuppet import FoxPuppet
+
+
+class TestBrowserModel(object):
+
+    def test_new_private_window(self, selenium):
+        """Tests opening a new private browsing window via menu"""
+        foxpuppet = FoxPuppet(selenium)
+        with foxpuppet.windows.wait_for_new_window():
```

Comment:
@whimboo I agree.

@jrbenny35 Thanks for the note. What Henrik means is that you shouldn't have to do this in every test. This is a new context manager I wrote. Not to manage the scope of the test but to manage the wait for the window to open. It does need to be cleaned up so that the test writer does not have to worry about adding the wait.